### ISSUE TITLE
I-ALiRT: codice lo algorithm

### ISSUE THREAD

**Issue by laspsandoval** [IMAP-Science-Operations-Center/imap_processing#1108]
### Requirements:

> **Specify: (optional)**

### Algorithm Code Information:
> Input data:
> > **Specify details:**
>
> Algorithm steps/psuedocode:
> > **Specify details:**
>
> Output data:
> > **Specify details:**

### Code:

```python
# Code
```

**Comment by tech3371** [comment]
I believe this is blocked by new SCI-LUT upload to onboard by CoDICE team. New SCI-LUT details been reviewed by project CCB. This task is blocked by that. Then if and when SDC receive new SCI-LUT, it may be blocked for a bit when we need to create the new SCI-LUT json.

**Comment by laspsandoval** [comment]
Instrument team approved that CoDICE-Lo data may be released. There was a single data point out of ~10^6 that is slightly off. A ticket was created in response, but marked as very low priority.
